markdown/blog/2023-mentorship-summary.md

```diff
@@ -0,0 +1,87 @@
+
+It's important to mention that we evaluated both the mentees and the mentors during the phases. This approach allows us to provide feedback to mentors for improvement and better mentorship experiences in the future.
+
+
+#### Projects success
+
+During the program, eight projects ranging from engineering to design and writing got some pretty great help. Let's shine a light on some of these projects:
+
+- **[Website UI Kit design/dev project](https://github.com/asyncapi/design-system/issues/4)**: Thanks to the success of this program, the AsyncAPI website got a design system, which is something we really wanted. Now whenever someone works on the website, there's a consistent way of doing things design-wise. 
+- **[Simulator Desktop Application](https://github.com/asyncapi/community/issues/691)**:  The AsyncAPI simulator, which started as a project in GSoC '21 and was just a command-line tool, now has a version that works on desktops. Now it's easier for people to use it right from their computers. 
+- **[Documenting how different protocols work with AsyncAPI](https://github.com/orgs/asyncapi/discussions/533)**: A group of four candidates embarked on a detailed project documenting the interaction between various protocols and AsyncAPI. Thanks to the program, they developed and published two series of tutorials covering how to use AsyncAPI with the [Kafka]([https://www.asyncapi.com/docs/tutorials/websocket](https://www.asyncapi.com/docs/tutorials/websocket)) and [Websocket]([https://www.asyncapi.com/docs/tutorials/websocket](https://www.asyncapi.com/docs/tutorials/websocket)) protocols.
+- **[Add support for translations](https://github.com/asyncapi/website/issues/267)**: Thanks to the implementation of the i18n, the AsyncAPI website now supports translation. It will be easier to add support for multiple languages on the website. 
+- **[Rewrite this template and NodeJS WS template](https://github.com/asyncapi/nodejs-template/issues/133)**: We overhauled the template by the program's success, transitioning from Nunjucks to the React template engine. Completing the switch modernizes our approach and designates React as the default engine moving forward, signaling a new era of template design.
+
+> Note: Each project chosen for the program was on the receiving end of profound contributions during this phase. For brevity, we’ve highlighted only a selection.
+
+#### Mentees' Successes  
+
+While all the selected projects received outstanding contributions from the mentees, I'm pleased to share some of the successes experienced by the mentees who participated in this program after its completion.
+
+- **Path to Maintainership**: After the program, some mentees, including Maya, Ansh, Sambhav, and Rishi, became maintainers of certain projects due to the knowledge they gained during the mentorship program.
```

Comment:
more names, Vaishnavi - Rohit and Joy

how about we make it more explicit? give them bigger shout out in the article

so have sub-bullet points, naming them all and what maintainer duties they took up?

- Maya - became designer maintainer in website
- Ansh - became committer maintainer in website
- Sambhav - became code triager maintainer in website
- Rishi - became maintainer in nodejs-template
- Vaishnavi - became docs triager maintainer in website
- Rohit - became docs triager maintainer in website
- Joy - became docs triager maintainer in website

linking to their profiles, and to repositories?

also in the other section you could mention who decided to me mentor this year - like explicitly

cc @alequetzalli